src/Config/IConfigParams.ts

```diff
@@ -6,9 +6,10 @@ export interface IConfigParams {
   claiming?: boolean;
   economicStrategy?: IEconomicStrategy;
   logger?: ILogger | null;
+  maxRetries?: number;
   ms?: any;
   password?: any;
-  providerUrl: string;
+  providerUrls: string[];
```

Comment:
we still need the `providerUrl`, probably as a getter function, or we completely remove it from `cli` and `dapp`
Although Ithink using the `core` to handle the `active providerUrl` is better

Reply:
@adibas03 Good point `activeProviderUrl` should be exposed.